Triggers.mediawiki

```diff
@@ -0,0 +1,78 @@
+<pre>
+  NEP: <to be assigned>
+  Title: Triggers for NeoContract
+  Author: Erik Zhang <erik@neo.org>
+  Type: Standard
+  Status: Draft
+  Created: 2017-10-16
+</pre>
+
+==Abstract==
+
+Trigger is a mechanism for triggering execution for smart contracts. This NEP defines 4 kinds of triggers, they are <code>Verification</code>, <code>VerificationR</code>, <code>Application</code> and <code>ApplicationR</code>.
+
+==Motivation==
+
+A blockchain that provides smart contract system should provide multiple triggers for the smart contracts running on it, makes them to function in different contexts.
+
+==Rationale==
+
+Currently, we have two kinds of triggers in NeoContract: <code>Verification</code> and <code>Application</code>. These two triggers make smart contracts to be able to verify the transactions and modify the states of the blockchain.
```

Comment:
"These two triggers make smart contracts to be able to verify the transactions"

should maybe be changed to:

"These two triggers make smart contracts able to verify transactions"